tox.ini

```diff
@@ -1,22 +1,27 @@
 [tox]
-envlist = py27
+envlist = py27, py34, py35, py36
+recreate = True
```

Comment:
Do you know why this option is necessary? It seems as though this force recreates the virtual environment.